homeassistant/components/knx/services.yaml

```diff
@@ -36,18 +36,18 @@ read:
         text:
 event_register:
   name: "Register knx_event"
-  description: "Add or remove single group address to knx_event filter for triggering `knx_event`s. Only addresses added with this service can be removed."
+  description: "Add or remove group addresses to knx_event filter for triggering `knx_event`s. Only addresses added with this service can be removed."
   fields:
     address:
       name: "Group address"
-      description: "Group address that shall be added or removed."
+      description: "Group address(es) that shall be added or removed. Lists are allowed."
       required: true
       example: "1/1/0"
       selector:
         text:
```

Comment:
In that case, this should not be a `text` selector anymore...

Reply:
👍 changed to `object:`